### ISSUE TITLE
Enhance the README

### ISSUE THREAD

**Issue by KaizenGirl1111** [KaizenGirl1111/GSGrihSangini#11]
This issue accepts PR like-
1. Grammar corrections in README
2. Animations, Graphics in README(relevant)
3. Refined lines
4. Any other enhancement

**Comment by darkling10** [comment]
The Readme file contains some unnecessary information which is created by create-react-app command.Please assign me this issue

**Comment by Aditya3403** [comment]
I am GSSOC'22 participant .I would like to contribute on this issue .Kindly assign it to me.

**Comment by purnimakatiyar** [comment]
I am GSSOC'22 participant .I would like to contribute on this issue .Can you please assign me?

**Comment by KaizenGirl1111** [comment]
assigned to @darkling10 for now, @purnimakatiyar @Aditya3403 and @imnrb you all are free to create new easy issues over the repo for now! looking forward to see issues from you:)

**Comment by KaizenGirl1111** [comment]
@darkling10 any updates? Otherwise it'll be assigned to @purnimakatiyar @Aditya3403 @Prajnaprabhu3 if you couldn't complete it ^_^

**Comment by Prajnaprabhu3** [comment]
Hey, I can take this up. I am willing to work on it

**Comment by KaizenGirl1111** [comment]
@Prajnaprabhu3 are you on it?

**Comment by Prajnaprabhu3** [comment]
Yes, I can work on this. Since I had not got a confirmation, I haven't started yet.

**Comment by prernadabi23** [comment]
Hello, @KaizenGirl1111 I would like to work on this issue. Can I take on this?
